### PR TITLE
Add readme file for running e2e test

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```bash
+$ export KUBECONFIG="[YourPath]/.kube/config"
+```
+
+Run e2e_piot
+```bash
+$ make e2e_pilot 
+```
+
+Replace e2e_pilot with other targets accordingly if necessary. The detail can
```
Comment (rkpagadala) [inline tests/README.md]: Change to 
The detail can be found in tests/istio.mk

Comment (wattli) [reply tests/README.md]: done

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+To run e2e test:
+
+```bash
+$ pwd
+.../src/istio.io/istio
+```
+
+Export KUBECONFIG to your kube config file, e.g.,
+```bash
+$ export KUBECONFIG="[YourPath]/.kube/config"
```
Comment (ldemailly) [inline tests/README.md]: that's optional no ?

**Pull request merged**